Power User identified issues 1

SusanRatiLane opened the pull request:
Proposed text to address the following small issues:

Fixes #1449 Screen1 Documentation
Fixes #1450 Screen Development Guidance
Fixes #1452 Procedure Blocks
Fixes #1451 Maps Component Marker
Fixes #1459 Texting Component

Change-Id: I572cc17f088ae443a45c6126298f3fdcb46ba46a

ewpatton commented:
Also requesting language review from @kdclang @stezelMIT @hnichols9560

ewpatton commented on appinventor/docs/reference/blocks/procedures.html:
```diff
@@ -210,8 +210,15 @@ <h4>Procedure Blocks</h4>
+    and type to change it. Procedure names in an app must be unique. App Inventor will not let you define two procedures
+    on the same screen with the same name. You can rename a procedure at any time while you are building the app, by
+    changing the label in the block. App Inventor will automatically rename the associated call blocks to match.</p>
+<p>Java kewords cannot be used as procedure names. <a href="https://en.wikipedia.org/wiki/List_of_Java_keywords">Here</a>
```
We might want to add `target="_blank"` here to open a new tab.

SusanRatiLane commented:
I believe I've addressed all the feedback.

> procedures.html
> In most curriculum docs, we use "input parameters" over arguments, but either one is ok.

I opted to leave it as "arguments," mostly because I was concerned about introducing new grammatical errors. If it's a problem, I can change it.

> There was discussion in an email thread about adding Any Component for Markers automatically when a Map component is added to an app. Did that happen?

Yes. That was done, so the instructions for adding an invisible marker are now obsolete. Removed.
